File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) {
    long long int rev=0;
    long long int t=x;
    while(x>0)
    {
        long long int z=x%10;
        rev=rev*10+z;
        x=x/10;
    }
    return (rev==t);
    
}

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int left=0;
        String s=String.valueOf(x);
        int n=s.length();
        int right=n-1;
        while(left<right){
            if(s.charAt(left)!=s.charAt(right)){
                return false;
            }
            left++;
            right--;
        }
        return true;
    }
}